### ISSUE TITLE
Directly inheriting from ActiveRecord::Migration is not supported anymore

### ISSUE THREAD

**Issue by matissg** [605data/strongbolt#31]
It seems to me there is some error in the gem as when after I've done Strongbolt install I do `rails db:migrate` and get this error:
```
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "strongbolt_capabilities" does not exist
LINE 8:                WHERE a.attrelid = '"strongbolt_capabilities"...
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod,
                     c.collname, col_description(a.attrelid, a.attnum) AS comment
                FROM pg_attribute a
                LEFT JOIN pg_attrdef d ON a.attrelid = d.adrelid AND a.attnum = d.adnum
                LEFT JOIN pg_type t ON a.atttypid = t.oid
                LEFT JOIN pg_collation c ON a.attcollation = c.oid AND a.attcollation <> t.typcollation
               WHERE a.attrelid = '"strongbolt_capabilities"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:85:in `async_exec'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:85:in `block (2 levels) in query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:46:in `block in permit_concurrent_loads'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/concurrency/share_lock.rb:185:in `yield_shares'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:45:in `permit_concurrent_loads'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:84:in `block in query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:612:in `block (2 levels) in log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:611:in `block in log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:603:in `log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:83:in `query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql_adapter.rb:769:in `column_definitions'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:166:in `columns'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/schema_cache.rb:67:in `columns'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/schema_cache.rb:73:in `columns_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:451:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attributes.rb:233:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attribute_decorators.rb:50:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:446:in `load_schema'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:349:in `attribute_types'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attribute_methods.rb:179:in `has_attribute?'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/inheritance.rb:55:in `new'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:120:in `block in from_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:119:in `map'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:119:in `from_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:101:in `block in default_capabilities='
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `inject'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `default_capabilities='
/vagrant/adsales/config/initializers/strongbolt.rb:33:in `block in <top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt.rb:112:in `setup'
/vagrant/adsales/config/initializers/strongbolt.rb:1:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `block in load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:655:in `block in load_config_initializer'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/notifications.rb:168:in `instrument'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:654:in `load_config_initializer'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:612:in `block (2 levels) in <class:Engine>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:611:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:611:in `block in <class:Engine>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `instance_exec'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `run'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:59:in `block in run_initializers'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:48:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:48:in `tsort_each_child'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:58:in `run_initializers'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:353:in `initialize!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/railtie.rb:185:in `public_send'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/railtie.rb:185:in `method_missing'
/vagrant/adsales/config/environment.rb:5:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `block in require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:329:in `require_environment!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:445:in `block in run_tasks_blocks'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands/rake/rake_command.rb:21:in `block in perform'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands/rake/rake_command.rb:18:in `perform'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/command.rb:46:in `invoke'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands.rb:16:in `<top (required)>'
bin/rails:4:in `require'
bin/rails:4:in `<main>'
PG::UndefinedTable: ERROR:  relation "strongbolt_capabilities" does not exist
LINE 8:                WHERE a.attrelid = '"strongbolt_capabilities"...
                                          ^
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:85:in `async_exec'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:85:in `block (2 levels) in query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:46:in `block in permit_concurrent_loads'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/concurrency/share_lock.rb:185:in `yield_shares'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:45:in `permit_concurrent_loads'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:84:in `block in query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:612:in `block (2 levels) in log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:611:in `block in log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:603:in `log'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql/database_statements.rb:83:in `query'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/postgresql_adapter.rb:769:in `column_definitions'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/abstract_adapter.rb:166:in `columns'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/schema_cache.rb:67:in `columns'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/connection_adapters/schema_cache.rb:73:in `columns_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:451:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attributes.rb:233:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attribute_decorators.rb:50:in `load_schema!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:446:in `load_schema'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/model_schema.rb:349:in `attribute_types'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/attribute_methods.rb:179:in `has_attribute?'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.1.1/lib/active_record/inheritance.rb:55:in `new'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:120:in `block in from_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:119:in `map'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/capability.rb:119:in `from_hash'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:101:in `block in default_capabilities='
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `inject'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt/configuration.rb:100:in `default_capabilities='
/vagrant/adsales/config/initializers/strongbolt.rb:33:in `block in <top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/strongbolt-0.3.11/lib/strongbolt.rb:112:in `setup'
/vagrant/adsales/config/initializers/strongbolt.rb:1:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `block in load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:286:in `load'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:655:in `block in load_config_initializer'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/notifications.rb:168:in `instrument'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:654:in `load_config_initializer'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:612:in `block (2 levels) in <class:Engine>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:611:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/engine.rb:611:in `block in <class:Engine>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `instance_exec'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `run'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:59:in `block in run_initializers'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:48:in `each'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:48:in `tsort_each_child'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/initializable.rb:58:in `run_initializers'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:353:in `initialize!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/railtie.rb:185:in `public_send'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/railtie.rb:185:in `method_missing'
/vagrant/adsales/config/environment.rb:5:in `<top (required)>'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `block in require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:329:in `require_environment!'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/application.rb:445:in `block in run_tasks_blocks'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands/rake/rake_command.rb:21:in `block in perform'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands/rake/rake_command.rb:18:in `perform'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/command.rb:46:in `invoke'
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/railties-5.1.1/lib/rails/commands.rb:16:in `<top (required)>'
bin/rails:4:in `require'
bin/rails:4:in `<main>'
Tasks: TOP => db:migrate => environment
(See full trace by running task with --trace)
```
There is one more question. I'm using rails 5.1. and was wondering should the migration look more like this?
```
class CreateStrongboltTables < ActiveRecord::Migration[5.1]
revert do
  def change
    create_table :strongbolt_capabilities do |t|
      t.string   :name, null: false
      t.string   :description, null: false
      t.string   :model, null: false
      t.string   :action, null: false
      t.string   :attr, null: false
      t.boolean  :require_ownership, default: false, null: false
      t.boolean  :require_tenant_access, default: false, null: false
      t.timestamps
    end

    create_table :strongbolt_roles do |t|
      t.string   :name, null: false
      t.integer  :parent_id, index: true
      t.integer  :lft, index: true
      t.integer  :rgt, index: true
      t.string   :description
      t.timestamps
    end

    create_table :strongbolt_user_groups do |t|
      t.string :name, null: false
      t.text :description
      t.timestamps
    end

    create_table :strongbolt_user_groups_users do |t|
      t.belongs_to :user_group, index: true, null: false
      t.belongs_to :user, index: true, null: false
    end

    create_table :strongbolt_roles_user_groups do |t|
      t.belongs_to :user_group, index: true, null: false
      t.belongs_to :role, index: true, null: false
    end

    create_table :strongbolt_capabilities_roles do |t|
      t.belongs_to :role, index: true, null: false
      t.belongs_to  :capability, index: true, null: false
    end

    create_table :strongbolt_users_tenants do |t|
      t.belongs_to  :user, index: true, null: false
      t.integer  :tenant_id, index: true, null: false
      t.string   :type, index: true, null: false
    end

    add_index :strongbolt_user_groups_users, [:user_group_id, :user_id], unique: true
  end
end
end
```
As I understand there has to be `strongbolt_` prefix for all tables.

**Comment by tobithiel** [comment]
See 605data/strongbolt_example#3

**Comment by matissg** [comment]
@tobithiel OK, now I see where the problem is. Sure I can switch to some other Rails 5 version - which one do you recommend? 
I've studied your gem's code and googled what was available and so far it seems to be what I really need for my project. Model level authorization seems to be the right thing :)

**Comment by tobithiel** [comment]
The error seems to have been introduced in Rails 5.1.0, so 5.0.3 should work fine afaik.
I'm glad you're interested in Strongbolt and find it useful :)

**Comment by matissg** [comment]
I downgraded to 5.0.3. however the error stays:
```
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "strongbolt_capabilities" does not exist
LINE 8:                WHERE a.attrelid = '"strongbolt_capabilities"...
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod,
             (SELECT c.collname FROM pg_collation c, pg_type t
               WHERE c.oid = a.attcollation AND t.oid = a.atttypid AND a.attcollation <> t.typcollation),
                     col_description(a.attrelid, a.attnum) AS comment
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"strongbolt_capabilities"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.0.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:88:in `async_exec'

# more lines follow
```

**Comment by tobithiel** [comment]
Did you rerun `rails g strongbolt:install && rake db:migrate`, so the tables get created?

**Comment by matissg** [comment]
Yes, this is what I've got:
```
ubuntu@ubuntu-xenial:/vagrant/myproject$ rails g strongbolt:install && rake db:migrate
/home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.0.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:88:in `async_exec': PG::UndefinedTable: ERROR:  relation "strongbolt_capabilities" does not exist (ActiveRecord::StatementInvalid)
LINE 8:                WHERE a.attrelid = '"strongbolt_capabilities"...
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod,
             (SELECT c.collname FROM pg_collation c, pg_type t
               WHERE c.oid = a.attcollation AND t.oid = a.atttypid AND a.attcollation <> t.typcollation),
                     col_description(a.attrelid, a.attnum) AS comment
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"strongbolt_capabilities"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
        from /home/ubuntu/.rvm/gems/ruby-2.3.3/gems/activerecord-5.0.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:88:in `block in query'
```

**Comment by tobithiel** [comment]
We'll need the full stacktrace to be able to see what's going on.

**Comment by matissg** [comment]
No need :) I shut down my Ubuntu, then deleted /config/initializers/strongbolt.rb Then started my Ubuntu and run `rails g strongbolt:install && rake db:migrate` and it worked now. Thank you for help so far!

**Comment by tobithiel** [comment]
I've just pushed a new commit which should make the setup compatible with the newest Rails 5.2.X versions.
Let me know if that works for you, then I'll push out a new version (until then use the master branch instead of the published version 0.3.11).

**Comment by matissg** [comment]
@tobithiel Thank you, it seems to be working now on Rails 5.1.1
